gallery.php

```diff
@@ -0,0 +1,84 @@
+<?php
+require_once ('src/DBconnector.php');
```

Comment:
the whole point of an autoloader is you dont need to require anything except the autoloader

Reply:
done